### PR TITLE
Add missing packages from Symbiflow/conda-packages

### PR DESCRIPTION
**Pull request by tjurtsch**
These package recipes are used by e.g. sv-tests flow.

### CODE REVIEW

**Comment by tjurtsch** [comment]
I also removed some Travis-related variables from recipes.

**Review by PiotrZierhoffer — CHANGES_REQUESTED** [review]

File: sim/uhdm-integration-verilator/condarc [hunk 1]
```diff
@@ -0,0 +1,2 @@
+channels:
+  - conda-forge
```
Comment (ajelinski) [inline sim/uhdm-integration-verilator/condarc:2]: Dump the whole `condarc` file. `conda-forge` won't be required to install `surelog-uhdm` and `verilator-uhdm` after changing their `condarc` files.

File: syn/yosys-uhdm/condarc [hunk 2]
```diff
@@ -0,0 +1,2 @@
+channels:
+  - conda-forge
```
Comment (ajelinski) [inline syn/yosys-uhdm/condarc:2]: Add `- defaults` as the first channel. This way the built package won't require `conda-forge` channel during installation.

File: pnr/odin_II/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+{% set version = '%s_%04i_%s'|format(GIT_DESCRIBE_TAG|replace('v','') or '0.X', GIT_DESCRIBE_NUMBER|int, GIT_DESCRIBE_HASH or 'gUNKNOWN') %}
```
[the remark this answers is not in the document]
Comment (tjurtsch) [reply pnr/odin_II/meta.yaml:1]: AFAIK yes, @ajelinski ?

File: syn/yosys-uhdm/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    - gperftools
+    - flex
+    - libunwind
+    - swig
+    - ncurses
+test:
+    requires:
+        - ncurses
+        - readline
+        - libffi
```
Comment (ajelinski) [inline syn/yosys-uhdm/meta.yaml:34]: With 99% certainty I'd say those 3 packages should be in `requirements: run` section if they are required to successfully call `yosys-uhdm --version`. Please add such section and remove this whole `requires`.

File: sim/uhdm-integration-yosys/condarc [hunk 5]
```diff
@@ -0,0 +1,2 @@
+channels:
+  - conda-forge
```
Comment (ajelinski) [inline sim/uhdm-integration-yosys/condarc:2]: Dump the whole `condarc` file. `conda-forge` won't be required to install `surelog-uhdm` and `yosys-uhdm` after changing their `condarc` files.

File: sv-front/slang/condarc [hunk 6]
```diff
@@ -0,0 +1,2 @@
+channels:
+  - conda-forge
```
[the remark this answers is not in the document]
Comment (ajelinski) [reply sv-front/slang/condarc:2]: Definitely. Please add `defaults` channel as in other `condarc` cases. There are no `litex-hub` dependencies so this is the only channel I'd add.

[the remark this answers is not in the document]
Comment (ajelinski) [reply sv-front/slang/condarc:2]: I'd leave `condarc`s of `odin_II` and `surelog` as they are. The former requires `libunwind` to run and the latter `gtk3`. Both of those are only available in the `conda-forge` channel.

File: pnr/odin_II/meta.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (ajelinski) [reply pnr/odin_II/meta.yaml:1]: We can remove it and use such lines for version in all git-based recipes (so all of the recipes added in this PR):
```
  # Use `conda-build-prepare` before building for a better version string.
  version: 0.0
```
However it's not like these `{% set version ...` do any harm and they will at least set some version if anyone wants to build the recipe without `conda-build-prepare`. It's only misleading if anyone checks how is the version string established.

Please do choose, I'm not sure which is better. Maybe leave the `set` line and only add the comment?

File: sim/uhdm-integration-verilator/condarc (hunk 1 above)
Comment (tjurtsch) [reply sim/uhdm-integration-verilator/condarc]: Dumped

File: syn/yosys-uhdm/meta.yaml (hunk 4 above)
Comment (tjurtsch) [reply syn/yosys-uhdm/meta.yaml]: Requirements moved

File: sim/uhdm-integration-yosys/condarc (hunk 5 above)
Comment (tjurtsch) [reply sim/uhdm-integration-yosys/condarc]: Dumped

File: syn/yosys-uhdm/condarc (hunk 2 above)
Comment (tjurtsch) [reply syn/yosys-uhdm/condarc:3]: Added

File: sv-front/surelog-uhdm/condarc [hunk 7]
```diff
@@ -0,0 +1,2 @@
+channels:
+  - conda-forge
```
[the remark this answers is not in the document]
Comment (tjurtsch) [reply sv-front/surelog-uhdm/condarc:3]: Added

File: sv-front/slang/condarc (hunk 6 above)
[the remark this answers is not in the document]
Comment (tjurtsch) [reply sv-front/slang/condarc:3]: Added defaults channel

File: sim/verilator-uhdm/condarc [hunk 8]
```diff
@@ -0,0 +1,2 @@
+channels:
+  - conda-forge
```
[the remark this answers is not in the document]
Comment (tjurtsch) [reply sim/verilator-uhdm/condarc:3]: Added

File: pnr/odin_II/meta.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (tjurtsch) [reply pnr/odin_II/meta.yaml:2]: Added comments explaining the role of conda-build-prepare

**Review by PiotrZierhoffer — APPROVED** [review]

**Review by ajelinski — APPROVED** [review]

**Pull request merged**